src/server/models/users.model.js

```diff
@@ -8,13 +8,10 @@ import mongoose from 'mongoose';
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
-  userId: String,
-  email: String,
+  email: { type: String, unique: true },
+  platform_id: String,
+  platform_type: Number,
   name: String,
-  age: Number,
```

Comment:
Are we removing these fields because they're unavailable? Some of these fields would be critical to the product overall. Do we plan on adding them once we have access to data? If so, add a todo:

TODO(yangwoo_yangwoo): Add support for field, region, ...